controller/src/test/java/io/pravega/controller/server/eventProcessor/ControllerEventProcessorsTest.java

```diff
@@ -98,9 +99,11 @@ public void testHandleOrphaned() {
         ControllerEventProcessorConfig config = ControllerEventProcessorConfigImpl.withDefault();
         EventProcessorSystem system = mock(EventProcessorSystem.class);
         EventProcessorGroup<ControllerEvent> processor = getProcessor();
+        EventProcessorGroup<ControllerEvent> mockProcessor = spy(processor);
 
+        doThrow(new CheckpointStoreException("host not found")).when(mockProcessor).notifyProcessFailure("host3");
         try {
-            when(system.createEventProcessorGroup(any(), any(), any())).thenReturn(processor);
+            when(system.createEventProcessorGroup(any(), any(), any())).thenReturn(mockProcessor);
         } catch (CheckpointStoreException e) {
             e.printStackTrace();
```

Comment:
remove this catch block.

Reply:
Thanks for reviewing, Removed catch block.